ConsoleApp1/Example.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace ConsoleApp1
```

Comment:
Please - move this to CourseApp (or add stylecop and gitlab build steps here manually